### ISSUE TITLE
SSL certificate not (yet) working

### ISSUE THREAD

**Issue by chartgerink** [bertelsmannstift/data-knowledge-hub#26]
**Does the issue concern a specific section? If yes, which one?**

None.

**What are you trying to improve and/or fix? Please describe.**

The [Data Knowledge Hub](https://data-knowledge-hub.com/) website is being built from GitHub pages now, and the DNS records are now updated. Because this change requires a new SSL certificate for HTTPS, the website is currently HTTP only until that is resolved.

![CleanShot 2024-10-02 at 08 55 42@2x](https://github.com/user-attachments/assets/28f68de3-129d-438c-a9ed-42af6d5ed192)

The DNS records are successfully updated, and GitHub recognizes this:

![CleanShot 2024-10-02 at 08 56 09@2x](https://github.com/user-attachments/assets/f6f422ff-05e1-4a17-bfed-24d94bc7ad47)

**Possible solutions or implementation suggestions**

Creating the SSL certificate is GitHub's job, and may take a moment. This issue is to track the problem so it stays on our radar. However, I expect it to be resolved in the course of the day. As [the GitHub documentation says](https://docs.github.com/en/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https#troubleshooting-certificate-provisioning-certificate-not-yet-created-error):
> If the check is successful, GitHub queues a job to request a TLS certificate from Let's Encrypt. On receiving a valid certificate, GitHub automatically uploads it to the servers that handle TLS termination for Pages. When this process completes successfully, a check mark is displayed beside your custom domain name.

> The process may take some time

I will check in throughout the day and provide updates.

**Comment by cathleenberger** [comment]
Thanks for documenting this. Here is an updated version of the errors that I see in the settings:
![image](https://github.com/user-attachments/assets/78856ffd-0893-426b-bc52-dd9203f6e3dc)

If you have any suggestions for how to best resolve this, we're all ears. :)
Thank you!

**Comment by chartgerink** [comment]
Thanks!

I did a tiny bit of digging on the DNS records and I discovered the issue:

```
www.data-knowledge-hub.com. 273	IN	CNAME	data-knowledge-hub.com.
data-knowledge-hub.com.	3276	IN	A	185.199.111.153
data-knowledge-hub.com.	3276	IN	A	185.199.108.153
data-knowledge-hub.com.	3276	IN	A	185.57.241.74
data-knowledge-hub.com.	3276	IN	A	185.199.110.153
data-knowledge-hub.com.	3276	IN	A	185.199.109.153
```

The specific issue is that they added all the records needed, but they did not remove the old A record: 
```
data-knowledge-hub.com.	3276	IN	A	185.57.241.74
```

Could you request the removal of the A record pointing to `185.57.241.74`? I think this is creating conflicting information that is confusing GitHub pages 😊

**Comment by cathleenberger** [comment]
Yes, indeed, we've removed the indicated record and the domain now works, however, it seems only without "www." Will run another check in settings and hopefully, we can resolve and close this issue then. Thank you for your support!

**Comment by chartgerink** [comment]
Ah yes, the good old cache is not helping on that! It may still remember when the certificate was not valid, and hence, keep telling you it is not valid now. 

I [ran a browser independent check](https://www.sslshopper.com/ssl-checker.html#hostname=data-knowledge-hub.com), to circumvent the local cache (certificate memory), and it seems to recognize it without the `www` - happy to debug more in case the issue persists 😊

![CleanShot 2024-10-08 at 09 03 43@2x](https://github.com/user-attachments/assets/d9d4a775-2188-483a-b729-d036ec288f81)

**Comment by cathleenberger** [comment]
Yes, it works without www but not with www. That issue seems to persist across browers and deleted cache history. Not a major obstacle, but something to revist for usability.

**Comment by cathleenberger** [comment]
Closing this issue for now, as the SSL certificate problem has been fixed. Should the www. issue continue in the future we may open up another issue to keep track of suggested fixes. Thank you very much for engaging with this!
